### PR TITLE
Adding favicon to examples

### PR DESCRIPTION
**Pull request by RafidMuhymin**
## Changes

Added favicon to all starter templates. Used the SVG version as the default favicon and the ico version as an alternate.

### CODE REVIEW

**Comment by natemoo-re** [comment]
This is looking great, thank you so much for contributing! I will circle back to this shortly and get it merged.

**Review by natemoo-re — APPROVED** [review]
These are great changes! Thank you so much for contributing! 🎉

**Review by FredKSchott — COMMENTED** [review]
Given that these are just example starter repos, I'm always in favor of reducing complexity / number of moving parts. I'd suggest not adding an SVG favicon for any of these examples and strictly sticking to the `ico` format. @natemoo-re what do you think? I'll leave to you to resolve, just wanted to add my 2 cents!

**Comment by natemoo-re** [comment]
@FredKSchott Good point, I agree. Compatibility > shiny new tech. I can remove the SVG favicon in this PR!

**Comment by FredKSchott** [comment]
got some time now, will merge and then remove SVG favicons

**Comment by RafidMuhymin** [comment]
@FredKSchott Should I remove the SVG favicons?

**Comment by FredKSchott** [comment]
Already done! Thanks for the offer though

File: examples/docs/src/layouts/Main.astro [hunk 1]
```diff
@@ -23,12 +23,10 @@ const githubEditUrl = `https://github.com/USER/REPO/blob/main/${currentFile}`
     <title>{content.title}</title>
     
     <link rel="stylesheet" href="/index.css" />
-    <link rel="stylesheet" href="/theme.css" />
-    <link rel="stylesheet" href="/code.css" />
```
Comment (MarcusOtter) [inline examples/docs/src/layouts/Main.astro:27]: Hmm, this undid my PR https://github.com/snowpackjs/astro/pull/1062 and broke the example project

File: examples/snowpack/src/pages/tutorials/react.md [hunk 2]
```diff
@@ -118,7 +118,7 @@ The `mount` configuration changes where Snowpack looks for and builds files. Eve
 ```diff
   mount: {
 -   /* ... */
-+   // directory name: 'build directory',
```
Comment (MarcusOtter) [inline examples/snowpack/src/pages/tutorials/react.md:121]: I think the comma makes sense, I believe it's expected that removing the `//` will produce working code.

File: examples/snowpack/src/pages/tutorials/svelte.md [hunk 3]
```diff
@@ -172,7 +172,7 @@ The `mount` configuration changes where Snowpack scan for and builds files. Head
 
   mount: {
 -   /* ... */
-+   // directory name: 'build directory',
```
Comment (MarcusOtter) [inline examples/snowpack/src/pages/tutorials/svelte.md:175]: Same as above

**Review by MarcusOtter — COMMENTED** [review]
Sorry, I missed reviewing this PR earlier! It did have a breaking change though, probably forgot to rebase before creating the PR 😄 No worrries though, I'll add it back in.

**Pull request merged**